doc/nrf/ug_radio_fem.rst

```diff
@@ -127,12 +133,57 @@ This feature uses a model to compensate the FEM gain for the following external
 The model assumes that the FEM supply voltage is constant.
 To provide the value of this voltage to the MPSL subsystem, use the :kconfig:option:`CONFIG_MPSL_FEM_POWER_VOLTAGE` option.
 
+This model is enabled by default, provided that nRF21540 GPIO/SPI is enabled and selected. Refer to :ref:`ug_radio_fem_nrf21540_spi_gpio` for instructions how to do that.
+
+Although the built-in model significantly improves the performance of nRF21540 in varying external conditions, in certain scenarios it might be desirable to disable it, e.g. to save memory. To disable the model and the resulting compensation, set :kconfig:option:`CONFIG_MPSL_FEM_POWER_MODEL` to ``n``.
+
+The figures below summarize the performance improvement of nRF21540 when the built-in model is used. The data points in the figures are results of nRF21540 gain measurements performed in controlled, laboratory conditions. Each of the figures compares gain achieved by a nRF21540 device in two cases: when the built-in model is used to control nRF21540 gain settings, and when the model is disabled. All the external conditions the model compensates for, i.e. temperature, supply voltage, carrier frequency and input power are presented in the figures.
+
+The figures are divided into three groups, each of the group representing performance improvement over frequency and one other external condition for the two factory-precalibrated values of gain.
```

Comment:
It is worth commenting why for 20dB there results are too low for certain voltage (this is because of FEM physical limitations at these voltages)

Reply:
In terms of readability, do you prefer it to be explained here or closest to the related figure?

Reply:
I would suggest adding 
`
.. note::   For 20dB and low voltages a better compensation cannot be achieved due to nRF21540 hardware limitations.
`
After all figures